### ISSUE TITLE
Ajouter layers sélectionnables par type d'engin

### ISSUE THREAD

**Issue by glabmoris** [CIDCO-dev/PecheFantome#8]
Les utilisateurs veulent avoir une catégorisation des engins perdus par type d'engin

**Comment by Ddoiron-cidco** [comment]
les donnée que nous avons ne fournissent pas la description des types d'engin si ma mémoire est bonne.

si c'Est le cas il vas faloir fragmenter les donnée en diférente base de donnée soit une par type d'engin.

Pour implémenter la gestion des layer ca me prendrais une liste des type d'engin a lister.

**Comment by Ddoiron-cidco** [comment]
je vien de valider on as pas les type d'engin dans le fichier de merinov

**Comment by glabmoris** [comment]
Oui, on peut fragmenter le tout pour permettre différents layers. Au meilleur de mes souvenirs on avait cette valeur dans le fichier original de Merinov .

**Comment by glabmoris** [comment]
@Ddoiron-cidco 

![Screenshot from 2020-03-19 16-16-43](https://user-images.githubusercontent.com/9091120/77111473-cae6ff00-69fd-11ea-8593-2bb835310bbf.png)

 Je confirme que cette info est dispo dans le fichier xlsx de Merinov (docs/Merinov.....)
